test/utils/consul-stub.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
```

Comment:
I don't think it's necessary, no. As far as I know, mocha won't run up anything in the "./utils" folder by default, and this file shouldn't be using anything mocha specific.